src/components/screens/MainScreen.tsx

```diff
@@ -13,37 +24,85 @@ import classnames, {
   width,
   zIndex,
 } from 'classnames/tailwind'
+import openBlankTab from 'helpers/openBlankTab'
 
 const whatPeopleThinkWrapper = classnames(
   scale('scale-50', 'xs:scale-65', 'md:scale-100'),
   transitionProperty('transition-transform')
 )
-const mottoWrapper = classnames(width('w-64', 'xs:w-80', 'md:w-96'))
+const elementsWidth = width('w-64', 'xs:w-80', 'md:w-104')
+const buttonsWidth = width('w-64', 'xs:w-80', 'md:w-50')
 const container = classnames(
   display('flex'),
   flexDirection('flex-col'),
   alignItems('items-center'),
-  justifyContent('justify-center'),
   zIndex('z-20'),
   padding('px-4')
 )
 
-const buttonContainer = classnames(margin('my-10'))
+const downloadButtonsWrapper = classnames(
+  margin('my-4', 'xs:my-10'),
+  display('flex'),
+  flexDirection('flex-col', 'md:flex-row'),
+  gap('gap-4'),
+  alignItems('items-center'),
+  justifyContent('justify-center')
+)
+
+const bottomBlockWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  alignItems('items-center'),
+  gap('gap-y-2'),
+  margin('mt-6')
+)
 
 export default function () {
   return (
     <div className={container}>
-      <div className={mottoWrapper}>
+      <div className={elementsWidth}>
         <AppMotto />
       </div>
 
-      <div className={buttonContainer}>
-        <Button small title="Join the waitlist" />
+      <div className={downloadButtonsWrapper}>
+        <Button
+          bold
+          smallPaddings
+          fixedHeight="h-15"
+          fixedWidth={buttonsWidth}
+          fontSized="text-lg"
+          leftIcon={GetOnTestflight()}
+          onClick={() => openBlankTab(testFlightLink)}
+        />
+        <Button
+          bold
+          smallPaddings
+          fixedHeight="h-15"
+          fixedWidth={buttonsWidth}
+          fontSized="text-lg"
+          leftIcon={GetOnGooglePlay()}
+          onClick={() =>
+            openBlankTab(isAndroid ? androidPlayStore : webPlayStore)
+          }
+        />
       </div>
 
       <div className={whatPeopleThinkWrapper}>
         <WhatPeopleThink />
       </div>
+
+      <div className={bottomBlockWrapper}>
+        <BasicText color="text-blue-light">
+          Not part of the YC community?
+        </BasicText>
+        <Button
+          outline
+          smallPaddings
+          color="text-blue-light"
+          title="Request an invite"
```

Comment:
Not sure that it's good idea to pass button text as `title`. It's better to pass it as children, imo

Reply:
It's fine, we can change it later